Userland/Applications/PixelPaint/Layer.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include "Layer.h"
 #include "Image.h"
+#include "LibGfx/Painter.h"
```

Comment:
Please use `<LibGfx/Painter.h>` since you're not importing from the same directory. You'll also need to put it in the correct sorted position.